### DISCUSSION TITLE
Is it feasible to use the MLP model as a cost for crocoddyl?

### DISCUSSION THREAD

**Discussion by MuziYT** [loco-3d/crocoddyl#1281] [Q&A]
Hi, is it feasible to use a MLP model as a cost for crocoddyl?

**Comment by cmastalli** [comment]
> Hi, is it feasible to use a MLP model as a cost for crocoddyl?

Crocoddyl's solvers can handle any nonlinear, yet differentiable, cost function that depends on the state and control. When introducing an MLP-like cost, this is the basic consideration to make.

For more technical details, please read our papers.

**Comment by MuziYT** [comment]
Thank you very much， I will try it.



---- Replied Message ----
| From | Carlos ***@***.***> |
| Date | 07/19/2024 00:23 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [loco-3d/crocoddyl] Is it feasible to use the MLP model as a cost for crocoddyl? (Discussion #1281) |

Hi, is it feasible to use a MLP model as a cost for crocoddyl?

Crocoddyl's solvers can handle any nonlinear, yet differentiable, cost function that depends on the state and control. When introducing an MLP-like cost, this is the basic consideration to make.

For more technical details, please read our papers.
